comms/opencbm-plugin-xa1541/pkg-plist

```diff
@@ -0,0 +1,5 @@
+etc/opencbm.conf.d/10xa1541.conf
```

Comment:
Should it be a sample file?

Reply:
No, it just contains the installation path of the plugin